crypto/keyring/keyring.go

```diff
@@ -52,6 +52,8 @@ var (
 
 // Keyring exposes operations over a backend supported by github.com/99designs/keyring.
 type Keyring interface {
+	// Get the backend type used in the keyring config: "file", "os", "kwallet", "pass", "test", "memory".
```

Comment:
this is fine for master but for the backport we shouldn't break the interface.

Reply:
sg, I'll remove the backport label then

Reply:
you can backport, just in 0.45 code you will need to type cast to get the method